lib/shell.js

```diff
@@ -34,6 +34,8 @@ module.exports = function(cmd, args, options, callback) {
         }
       };
 
+  process.setMaxListeners(0);
```

Comment:
Well, in that case...  ;-)

Yeah, good call.  Not a band-aid.  If you want it to appropriate complain, set it to the expected concurrency * 2 (since each invocation attaches 2 listeners).